readme.md

```diff
@@ -136,10 +136,16 @@ This project uses [BATS](https://bats-core.readthedocs.io/en/latest/) for unit t
 
 ### ```-h|--help```
 
-**Usage**: ```./main.sh -```
+**Usage**: ```./main.sh -h```
 
 **Purpose**:  Prints help regarding the parameters to pass to this script
 
+### ```-co|--cluster-only```
+
+**Usage**: ```./main.sh --cluster-only```
+
+**Purpose**:  Skips install for pre-installed images
```

Comment:
I think this needs expanding a bit, what does it mean and what actually happens? Why would you ever install if pre installed?

Reply:
Yep.. expanded on it a bit